packages/fxa-content-server/app/scripts/views/sign_up_password.js

```diff
@@ -104,6 +120,20 @@ const SignUpPasswordView = FormView.extend({
   _doPasswordsMatch() {
     return this._getPassword() === this._getVPassword();
   },
+
+  _onConfirmPasswordKeyUp() {
+    this.checkPasswordsMatchDebounce();
+  },
+
+  _onConfirmPasswordBlur() {
+    this._checkPasswordsMatch();
+  },
```

Comment:
Does this mean the check is performed twice when I tab out of the confirm password field?  Once on the blur, and then again `DELAY_BEFORE_PASSWORD_CHECK_MS` after the last debounce call?

Reply:
Hmm yea, it does. Let me add a few checks to make this work a little smoother.